gengokit/gengokit_test.go

```diff
@@ -69,17 +69,13 @@ func TestNewTemplateExecutor(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	const goImportPath = "github.com/TuneLab/go-truss/gengokit"
+	const goImportPath = "github.com/TuneLab/go-truss/gengokit/general-service"
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
```
func newTemplateExecutor(dt deftree.Deftree, goImportPath, goPBImportPath string) (*templateExecutor, error) {
```

This is because the template executor needs to know where the `goImportPath` of the service
is as well as where the `goPBImportPath` is in order to template the imports in the generated service. For these tests though the I'm just saying they will be in the same place as that is the case by default. I could add a comment about each about this.